docs/community/governance.md

```diff
@@ -73,28 +73,82 @@ long experience with both the project and the larger ecosystem. When the core
+is elected by the Institutional and Funding Partner Advisory Council.
+
+The SC may be contacted at `napari-steering-council@googlegroups.com`.
+Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team. For a list of the current steering council see our [About Us](team) page.
+
+### Institutional and funding partners
+
+The SC is the primary leadership body for napari. No outside institution,
+individual or legal entity has the ability to own or control the project
+other than by participating in napari as contributors, core developers, and
+SC members. However, because institutions can be an important source of
+funding and contributions for the project, it is important to formally
+acknowledge institutional participation in the project. We call institutions
+recognized in this way Institutional and Funding Partners.
+
+Institutions become eligible to become an Institutional Partner by employing
+individuals who actively contribute to napari as part of their official
+duties, or by committing significant funding to napari, as determined by the
+SC. Once an institution becomes eligible to become an Institutional Partner,
+the SC must nominate and approve the institution as an Institutional and
+Funding Partner. At that time one individual from the Institutional and
+Funding Partner is expected to become the Institutional and Funding Partner
+Representative and serve on the Institutional and Funding Partner Advisory
+Council. The role of the Institutional and Funding Partner Advisory Council
+is to provide input on project directions and plans, and to elect one
+Institutional and Funding Partner Representative to hold the Institutional
+and Funding Partner seat on the SC.
+
+The Institutional and Funding Partners Advisory Council is expected to
+self-organize according to rules agreed to by the existing Institutional
+Funding Partners. This document does not prescribe how Institutional Funding
+Partners should elect their representative on the napari Steering Council.
+Institutional Funding Partners are expected to work together and with the
+napari community in good faith towards a common goal of improving napari for
+the broader scientific computing community.
+
+If at some point an existing Institutional and Funding Partner is no longer
+contributing any employees or funding, then a one-year grace period
+commences. If during this one-year period they do not contribute any
+employees or funding, then at the end of the period their status as an
+Institutional and Funding Partner will lapse, and resuming it will require
+going through the normal process for new Institutional and Funding Partners.
+If the Institutional and Funding Partner Representative on the SC is from an
+organization that loses its status as an Institutional Partner, that person
+will cease being a member of the SC and the remaining Institutional and
+Funding Partner Advisory Council members may choose a new Representative.
```

Comment:
This will happen immediately, or on the following january?

Reply:
I would argue immediately. I read the January thing is more of a prod to think about turning over, rather than as a date where *all* SC transactions have to happen.

Reply:
```suggestion
will cease being a member of the SC and the remaining Institutional and
Funding Partner Advisory Council members may choose a new Representative at their
earliest convenience.
```

Reply:
suggestion added